src/thematic/1_water/sfp-106-nl_waterpeil-peilbuis/02_onderwerp.Rmd

```diff
@@ -0,0 +1,37 @@
+# Onderwerp
+
+## Definities en afkortingen
+
+```{=html}
+<!--
+Definities van begrippen en afkortingen die noodzakelijk zijn om dit document te begrijpen en het protocol op een gepaste manier te kunnen uitvoeren.
+-->
+```
+-   **Peil:** De gemeten waarde van de top van de peilbuis naar het wateroppervlak in de peilbuis.
+    Soms wordt dit ook herleidt t.o.v. het maaiveld (m-mv), maar binnen het INBO is de afspraak dat het peil t.o.v. de top van de peilbuis genoteerd wordt.
+
+-   **Stijghoogte / drukhoogte:** Het potentieel peil van het wateroppervlak van grondwater, gemeten vanaf een bepaald niveau (bv. Tweede Algemene Waterpassing, TAW).
+    Doorgaans wordt de bodem van de peilbuis gebruikt.
+
+-   **TAW:** Tweede Algemene Waterpassing.
+    Referentiehoogte waar alle hoogtemetingen in België tegenover gebruikt worden.
+    Het gemiddeld zeeniveau bij laagwater te Oostende wordt gebruikt als nulpeil.
+
+## Doelstelling en toepassingsgebied
+
+```{=html}
+<!--
+Doelstelling: omschrijving in woorden waarvoor het protocol dient.
+Voorbeelden wat er met de gegevens kan gedaan worden.
+Probeer de doelstelling zo veel mogelijk generiek te beschrijven en vermijd projectspecifieke doelstellingen.
+Toepassingsgebied: bespreek hier onder welke condities het protocol kan gebruikt worden.
+-->
+```
+Deze richtlijn heeft betrekking op peilmetingen in een peilbuis, geplaatst volgens het protocol beschreven in de richtlijn SVP-104 (richtlijnen voor het plaatsen van een peilbuis)..
```

Comment:
Dus dit protocol is afhankelijk van https://github.com/inbo/protocolsource/pull/52. Dat betekent dat we eerst die PR moeten afwerken. Daarna kunnen we naar dat protocol verwijzen. Er is `protocolhelper::add_dependency()` om ons daarbij te helpen.